lago/plugins/vm.py

```diff
@@ -411,22 +423,39 @@ def copy_to(self, local_path, remote_path, recursive=True):
         with LogTask(
             'Copy %s to %s:%s' % (local_path, self.name(), remote_path),
         ):
-            with self._scp() as scp:
-                scp.put(
-                    files=local_path,
-                    remote_path=remote_path,
-                    recursive=recursive,
-                )
+            try:
+                with self._scp() as scp:
+                    scp.put(
+                        files=local_path,
+                        remote_path=remote_path,
+                        recursive=recursive,
+                    )
+            except OSError:
```

Comment:
This will catch other exceptions as well (such as missing permissions), which is OK.
Yet, we need to change `LagoCopyFilesToVMError`, because its message refers only to file which don't exist. I think that we can append the original error to the message that we output for the user.

```python
except OSError as e:
    raise(local_path, str(e))
```